datamodel/APC.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "APC",
+  "description": "Article Processing Charge status",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier for the vocabulary",
+      "default": 1,
```

Comment:
Would suggest removing these `default` properties from all the JSON Schema files, they serve no purpose and their meaning can confuse.